packages/opentelemetry-resources/test/Resource.test.ts

```diff
@@ -325,28 +337,27 @@ describe('Resource', () => {
     });
   });
 
-  describe('compatibility', () => {
-    it('should merge resource with old implementation', () => {
-      const resource = Resource.EMPTY;
-      const oldResource = new Resource190({ fromold: 'fromold' });
+  // describe('compatibility', () => {
+  //   it('should merge resource with old implementation', () => {
+  //     const resource = Resource.EMPTY;
+  //     const oldResource = new Resource190({ fromold: 'fromold' });
 
-      const mergedResource = resource.merge(oldResource);
+  //     const mergedResource = resource.merge(oldResource);
 
-      assert.strictEqual(mergedResource.attributes['fromold'], 'fromold');
-    });
+  //     assert.strictEqual(mergedResource.attributes['fromold'], 'fromold');
+  //   });
 
-    it('should merge resource containing async attributes with old implementation', async () => {
-      const resource = new Resource(
-        {},
-        Promise.resolve({ fromnew: 'fromnew' })
-      );
-      const oldResource = new Resource190({ fromold: 'fromold' });
+  //   it('should merge resource containing async attributes with old implementation', async () => {
+  //     const resource = new Resource({
+  //       attributes: { fromNew: Promise.resolve('fromnew') },
+  //     });
+  //     const oldResource = new Resource190({ fromold: 'fromold' });
 
-      const mergedResource = resource.merge(oldResource);
-      assert.strictEqual(mergedResource.attributes['fromold'], 'fromold');
+  //     const mergedResource = resource.merge(oldResource);
+  //     assert.strictEqual(mergedResource.attributes['fromold'], 'fromold');
 
-      await mergedResource.waitForAsyncAttributes?.();
-      assert.strictEqual(mergedResource.attributes['fromnew'], 'fromnew');
-    });
-  });
+  //     await mergedResource.waitForAsyncAttributes?.();
+  //     assert.strictEqual(mergedResource.attributes['fromnew'], 'fromnew');
+  //   });
+  // });
```

Comment:
hmm, this looks like a leftover.

Reply:
It was actually intentional as I wasn't sure if we'd remove the compatibility (and therefore tests) but I think its best if we do